### DISCUSSION TITLE
Is there a way to use template to make a find and replace?

### DISCUSSION THREAD

**Discussion by francisHugo** [SilentVoid13/Templater#1107] [Help] [answered]
Hi all,

what I am try to do is to set up a template, when I apply the template in the current note, it finds certain texts and then replace it with other texts.

For example, in the current notes, I have a tag `#inProgres`.  I would like to have a template, which can find it and replace it with `#done`, so i can assign a hotkey to the template, to make the modification of the tags easy.

Or, maybe there is another way to get it done. but with my limited Obsidian knowledge, I just came across to the magic plugin, template

Thanks

**Comment by Zachatoo** [comment, marked as answer]
Definitely, here's an example template of replacing all instances of `#tag1` with `#tag2` in the active file.

```js
<%*
const newContent = tp.file.content.replace("#tag1", "#tag2");
await app.vault.modify(tp.file.find_tfile(tp.file.title), newContent);
-%>
```

**Comment by francisHugo** [comment]
Thanks @Zachatoo 

This is exactly what I am looking for.  Thanks a lot. 

Best Regards,

Francis
